internal/bminventory/inventory.go

```diff
@@ -3378,7 +3378,7 @@ func (b *bareMetalInventory) V2DownloadHostIgnition(ctx context.Context, params
 		return common.GenerateErrorResponder(err)
 	}
 
-	return filemiddleware.NewResponder(installer.NewV2DownloadHostIgnitionOK().WithPayload(respBody), fileName, contentLength)
+	return filemiddleware.NewResponder(installer.NewV2DownloadHostIgnitionOK().WithPayload(respBody), fileName, contentLength, nil)
```

Comment:
Maybe worth using params.InfraEnvID to fetch the entity and add infraEnv.UpdatedAt also here? Or is it out of the scope for now?

Reply:
I don't think the infraenv has anything to do with the content of the host ignition, right?
Maybe host updated_at would be useful, but also I don't think it matters.

Reply:
Right, we use infraenv id just to fetch the host from db. But yeah, it's not that interesting.